site/docs/components/stepped-tracker/examples.mdx

```diff
@@ -20,6 +20,20 @@ You can add a label to the `TrackerStep` using the `StepLabel` component.
 
   </LivePreview>
 
+  <LivePreview componentName="stepped-tracker" exampleName="Compact" >
```

Comment:
The example doesn't load from storybook story but a duplicate in `site/src/examples`

Reply:
May need adjustment, so it will sit in the middle of example container.